org.pgadmin.pgadmin4.yml

```diff
@@ -0,0 +1,146 @@
+app-id: org.pgadmin.pgadmin4
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+rename-desktop-file: pgadmin4.desktop
+rename-icon: pgadmin4
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=session-bus
```

Comment:
you have to remove this too or explain why it is needed.

Reply:
I think it prompts you diffently on password storage, depending on this.

Will test, when I'm back at a pc.

Reply:
`--log-session-bus` and `--log-system-bus` are here for that.

(for system bus you need to grant a name for it to work)

Reply:
Something on the session bus seems to be needed for the masterpassword that recovers all your saved connections. Not sure how to parse the output of the log commands.

```
╰─ flatpak run --log-session-bus org.pgadmin.pgadmin4                                                                                                                                                                      ─╯
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.portal.Flatpak call org.freedesktop.DBus.Properties.Get at /org/freedesktop/portal/Flatpak
B47: <- :1.112 return from C2
C3: -> org.freedesktop.portal.Flatpak call org.freedesktop.DBus.Properties.Get at /org/freedesktop/portal/Flatpak
B48: <- :1.112 return from C3
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.portal.Flatpak call org.freedesktop.portal.Flatpak.Spawn at /org/freedesktop/portal/Flatpak
B50: <- :1.112 return from C3
B51: <- :1.112 signal org.freedesktop.portal.Flatpak.SpawnStarted at /org/freedesktop/portal/Flatpak
[3:1113/155438.468126:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Gtk-Message: 15:54:38.779: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:54:38.779: Failed to load module "pk-gtk-module"
Gtk-Message: 15:54:38.785: Failed to load module "canberra-gtk-module"
Gtk-Message: 15:54:38.785: Failed to load module "pk-gtk-module"
[3:1113/155438.795481:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[3:1113/155438.795509:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.NameHasOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C3
C4: -> org.freedesktop.portal.Desktop call org.freedesktop.DBus.Properties.Get at /org/freedesktop/portal/desktop
B571: <- :1.86 return from C4
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C3
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C4
C5: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C5
C6: -> org.freedesktop.portal.Desktop call org.freedesktop.portal.Settings.Read at /org/freedesktop/portal/desktop
C7: -> org.freedesktop.portal.Desktop call org.freedesktop.portal.Settings.Read at /org/freedesktop/portal/desktop
B576: <- :1.86 return from C6
B577: <- :1.86 return from C7
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.a11y.Bus call org.freedesktop.DBus.Properties.Get at /org/a11y/bus
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C2
*REWRITTEN*
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.NameHasOwner at /org/freedesktop/DBus
Filtering message due to arg0 org.freedesktop.secrets, policy: 0 (required 1)
B-1: <- org.freedesktop.DBus return from C3
*REWRITTEN*
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.ListActivatableNames at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C4
[43:1113/155443.009566:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.portal.IBus call org.freedesktop.DBus.Peer.Ping at /org/freedesktop/IBus
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B344: <- :1.59 return from C2
B-1: <- org.freedesktop.DBus return from C2
B-1: <- org.freedesktop.DBus return from C3
B-1: <- org.freedesktop.DBus return from C4
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.ListNames at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C3
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C4
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C-65536: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C-65535: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.pgadmin.pgadmin4
C-65534: -> org.freedesktop.DBus fake wildcarded AddMatch for org.pgadmin.pgadmin4
C-65533: -> org.freedesktop.DBus fake ListNames
B-1: <- org.freedesktop.DBus return from C1
B-1: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C-65536
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65535
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65534
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65533
C-65532: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C-65531: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C-65530: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C-65529: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C-65528: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65532
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65531
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65530
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65529
*SKIPPED*
B-1: <- org.freedesktop.DBus return from C-65528
*SKIPPED*
C2: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C2
C3: -> org.freedesktop.DBus call org.freedesktop.DBus.ListNames at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C3
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.RemoveMatch at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C4
C3: -> org.freedesktop.portal.IBus call org.freedesktop.IBus.Portal.CreateInputContext at /org/freedesktop/IBus
B345: <- :1.59 return from C3
C4: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C5: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
C6: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C7: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
C8: -> org.freedesktop.DBus call org.freedesktop.DBus.GetNameOwner at /org/freedesktop/DBus
B-1: <- org.freedesktop.DBus return from C4
B-1: <- org.freedesktop.DBus return from C5
B-1: <- org.freedesktop.DBus return from C6
B-1: <- org.freedesktop.DBus return from C7
B-1: <- org.freedesktop.DBus return from C8
C9: -> :1.59 call org.freedesktop.DBus.Properties.GetAll at /org/freedesktop/IBus/InputContext_17
B346: <- :1.59 return from C9
C10: -> :1.59 call org.freedesktop.DBus.Properties.Set at /org/freedesktop/IBus/InputContext_17
C11: -> :1.59 call org.freedesktop.IBus.InputContext.SetCapabilities at /org/freedesktop/IBus/InputContext_17
C12: -> :1.59 call org.freedesktop.IBus.InputContext.Reset at /org/freedesktop/IBus/InputContext_17
C13: -> :1.59 call org.freedesktop.IBus.InputContext.Reset at /org/freedesktop/IBus/InputContext_17
C14: -> :1.59 call org.freedesktop.DBus.Properties.Set at /org/freedesktop/IBus/InputContext_17
C15: -> :1.59 call org.freedesktop.IBus.InputContext.FocusIn at /org/freedesktop/IBus/InputContext_17
B349: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
C16: -> :1.59 call org.freedesktop.IBus.InputContext.Reset at /org/freedesktop/IBus/InputContext_17
C17: -> :1.59 call org.freedesktop.IBus.InputContext.FocusOut at /org/freedesktop/IBus/InputContext_17
C18: -> :1.59 call org.freedesktop.IBus.InputContext.FocusIn at /org/freedesktop/IBus/InputContext_17
C19: -> :1.59 call org.freedesktop.IBus.InputContext.FocusOut at /org/freedesktop/IBus/InputContext_17
B350: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B351: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B352: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B353: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B354: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B355: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B356: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B357: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
B358: <- :1.59 signal org.freedesktop.IBus.InputContext.UpdatePreeditTextWithMode at /org/freedesktop/IBus/InputContext_17
```

Reply:
probably `org.freedesktop.secrets` then, try it with a `--talk-name=`

Reply:
yeah, seems to work. it still complains on startup about system bus, but I think it's not the first app, where I ignore this.

should be good to go.